tests/gate_data.py

```diff
@@ -245,6 +245,18 @@ def ControlledPhaseShift(phi):
     return np.diag([1, 1, 1, np.exp(1j * phi)])
 
 
+def CPhase(phi):
+    r"""Controlled phase.
+
+    Args:
+        phi (float): rotation angle
+
+    Returns:
+        array: the two-wire controlled-phase matrix
+    """
+    return np.diag([1, 1, 1, np.exp(1j * phi)])
+
+
```

Comment:
Perhaps we could get away with not having this either? We're depending on `ControlledPhaseShift` in the test.
```suggestion
```

Reply:
Sure. I forgot about this one!